### PR TITLE
github: add an option to replace an existing ci file

### PR DESCRIPTION
**Pull request by noamcattan**
Fixes the following error:
image

By adding a new flag, --replace.

If the flag is not provided the error will be:
image

### CODE REVIEW

File: github.go [hunk 1]
```diff
@@ -9,12 +9,15 @@ import (
...
 
-	"ariga.io/gh-atlas/gen"
 	"github.com/cli/go-gh/pkg/repository"
 	"github.com/google/go-github/v49/github"
 	"golang.org/x/crypto/nacl/box"
+
+	"ariga.io/gh-atlas/gen"
 )
 
+const ACTION_FILE_PATH = ".github/workflows/ci-atlas.yaml"
```
Comment (dorav) [inline github.go:19]: this const can have a smaller scope inside the `AddAtlasYAML`, consider moving it inside the function

Comment (dorav) [reply github.go:19]: Also, naming-wise, I think we are using this convension?
https://stackoverflow.com/a/22689940/1708860

File: main_test.go [hunk 2]
```diff
@@ -218,14 +231,56 @@ func TestRunInitActionCmd(t *testing.T) {
...
 			cmd: &InitActionCmd{
 				DirPath: "migrations",
 				Driver:  "mysql",
 			},
 			prompt:  "invalid token\n",
 			wantErr: true,
 		},
+		{
+			name: "file exists",
+			client: &githubClient{
```
Comment (ronenlu) [inline main_test.go:245]: maybe add helper func: `createGHClient(repoSvc: Repositories)`.
and init the client using: `createGHClient(&mockService{})` or  `createGHClient(&mockService{getContentError: &github.ErrorResponse{Message: "Not Found"}})`

**Review by ronenlu — APPROVED** [review]
Nice!

**Pull request merged**